maquetado/scss/detail.scss

```diff
@@ -35,9 +40,37 @@
 .list-of-books {
   display: flex;
   flex-wrap: wrap;
+  margin-top: 100px;
 }
 
 .book-image {
   height: 280px;
   width: 200px;
 }
+
+.wolox-image {
+  height: 70px;
+  width: 200px;
+}
+
+.navbar {
+  align-items: center;
+  background-color: $white;
+  border-top: 5px solid $light-blue;
+  box-shadow: 5px 5px 10px 0 $black-05;
+  display: flex;
+  flex-direction: row;
```

Comment:
There is no need to set this as row. By default it works like that once you add `display: flex`